### PR TITLE
Add support for Python 3.5

### PR DESCRIPTION
**Pull request by maurizi**
Builds on top of #84

### CODE REVIEW

File: dev_requirements.txt [hunk 1]
```diff
@@ -2,3 +2,4 @@ coveralls==0.3
 coverage==3.6
 flake8==2.5.1
 django>=1.5
+unicodecsv==0.14.1
```
Comment (steventlamb) [inline dev_requirements.txt]: should probably be `unicodecsv>=0.14`

File: setup.py [hunk 2]
```diff
@@ -23,5 +23,5 @@
         "Framework :: Django",
         "License :: OSI Approved :: GNU General Public License (GPL)"
     ],
-    install_requires=['django>=1.5'],
+    install_requires=['django>=1.5', 'unicodecsv==0.14.1'],
```
Comment (steventlamb) [inline setup.py]: should probably be `unicodecsv>=0.14`

File: djqscsv/djqscsv.py [hunk 3]
```diff
@@ -172,15 +171,6 @@ def _validate_and_clean_filename(filename):
...
-def _safe_utf8_stringify(value):
-    if isinstance(value, six.string_types):
-        return value
-    elif isinstance(value, six.binary_type):
-        return value.encode('utf-8')
-    else:
-        return six.text_type(value).encode('utf-8')
-
-
 def _sanitize_unicode_record(field_serializer_map, record):
```
Comment (steventlamb) [inline djqscsv/djqscsv.py]: this should maybe be `_sanitize_record` or `_textify_record` now, it no longer does anything about unicode.

**Comment by steventlamb** [comment]
LGTM - is this planned for a major or minor version increase?

File: dev_requirements.txt (hunk 1 above)
Comment (maurizi) [inline dev_requirements.txt]: Changed in 351a114

File: setup.py (hunk 2 above)
Comment (maurizi) [inline setup.py]: Changed in 351a114

File: djqscsv/djqscsv.py (hunk 3 above)
Comment (maurizi) [inline djqscsv/djqscsv.py]: Good idea, I switched to `_sanitize_record` in  	60803a4

**Comment by maurizi** [comment]
> Is this planned for a major or minor version increase?

Since this library is fairly stable and this is the last major item on the "TODO" list, I figured I'd do a 1.0 release.

**Comment by ddohler** [comment]
I only have theoretical knowledge of unicode handling in Python 3 but the broad strokes here look right.

**Comment by jwalgran** [comment]
While working on other things I tried provisioning an OpenTreeMap VM with this branch by adding a git url to the requirements.txt file. I hit an error but will make another attempt at doing some regression testing in the context of a Python 2.7 app.

**Comment by zzzuzik** [comment]
+1

**Pull request merged**